test/fake/toolchainstatus_client.go

```diff
@@ -0,0 +1,72 @@
+package fake
+
+import (
+	"encoding/json"
+	"os"
+
+	crtapi "github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/testing"
+)
+
+type FakeToolchainStatusClient struct {
+	Tracker   testing.ObjectTracker
+	Scheme    *runtime.Scheme
+	namespace string
+	MockGet   func() (*crtapi.ToolchainStatus, error)
+}
+
+func NewFakeToolchainStatusClient(namespace string, initObjs ...runtime.Object) *FakeToolchainStatusClient {
+	clientScheme := runtime.NewScheme()
+	err := crtapi.SchemeBuilder.AddToScheme(clientScheme)
+	if err != nil {
+		log.Error(err, "Error adding to scheme")
+		os.Exit(1)
```

Comment:
are you sure you want to `os.Exit`? Should you call something like `t.Fatal` instead?

Reply:
It was a copy-paste. Did some small refactoring for that: https://github.com/codeready-toolchain/registration-service/pull/141/commits/7f1d753c9461c654e2fe787a9164e73c45d9225a